vipps-login-api-faq.md

```diff
@@ -271,13 +273,44 @@ Vipps Login from phone number are reserved for such special cases and needs to b
 enabled by Vipps for eligible sale units. If a merchant uses Vipps Login from phone number on
 webpages or in apps used by end-users (on their own device), access to the feature can be withdrawn.
 
-Merchants need to apply for access to Vipps Login from phone number separately by sending an email
-to accessuserinfo@vipps.no. In the email you should specify:
+Merchants need to apply for access to Vipps Login from phone number separately. 
+This is done by sending an email to accessuserinfo@vipps.no and supplying the following information:
+ * Merchant serial number (MSN) for the sale unit for which you will use Vipps Login (this can be retrieved from portal.vipps.no)
```

Comment:
```suggestion
 * Merchant serial number (MSN) for the sale unit for which you will use Vipps Login (this can be retrieved from [portal.vipps.no](https://portal.vipps.no))
```